### DISCUSSION TITLE
Welcome to MuseScore Discussions!

### DISCUSSION THREAD

**Discussion by Harmoniker1** [Howard-C/MuseScore#511] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PtJade-Ceramic** [comment]
I had owned the MuseScore account [@PtJade Ceramic](https://musescore.org/en/user/4582378).

As I am unwelcome in the MuseScore community in Chinese, I have deleted my account on 2025-08-26 and left the community.

I am trying to respond to your [request](https://musescore.org/zh-hans/node/381738#comment-1308462) but have trouble in logging in with the same electronic mail address, receiving the prompt `The website encountered an unexpected error. Please try again later.` If possible, let the administrator of the website merge my account and I am willing to update that post, or… I can just say sorry.
